### ISSUE TITLE
Tracking user access details: Usuario tutor general de sesión no puede ver detalle de los accesos de un usuario a un curso de la sesión

### ISSUE THREAD

**Issue by juancp-contidosdixitais** [chamilo/chamilo-lms#4981]
**Descripción**
Un tutor general de una sesión (session coach) no puede ver el detalle de los accesos de un estudiante (main/mySpace/access_details.php)

Esto es debido a que en unicamente se permite acceder a estos datos en los siguiente casos:

1. Administrador de la plataforma, administrador de sesiones, gestor de recursos humanos (DRH)
2. Cualquier usuario profesor en una plataforma con la opción de permitir crear cursos habilitada
3. Tutores del curso

https://github.com/chamilo/chamilo-lms/blob/9610bd60afeb12bdec0ffa0d9e11dd1b89e96353/main/inc/ajax/myspace.ajax.php#L12-L18

Por tanto un usuario asignado como tutor general de la sesión, que no es administrador en una la plataforma que no tiene activada la opción de permitir crear cursos a los profesores, nunca podrá ver el detalle de acceso de los usuarios de los cursos de esa sesión, incluso aunque sea agregado como tutor de cada curso. 

**Cómo reproducir**
Pasos para reproducir el problema:
1. Crear o localizar una sesión y asignar como tutor general a un usuario profesor no administrador
2. Verificar que está deshabilitada la opción de permitir crear cursos a los usuarios
3. Agregar alumnos a esa sesión y acceder al curso varias veces
4. Loguearse como el tutor, ir a informes, seleccionar uno de los alumnos que accediera al curso e ir a la herramienta "Detalles de acceso"

Detalle informe de un usuario alumno, se destaca acceso a la herramienta "Detalles de acceso"
![imagen](https://github.com/chamilo/chamilo-lms/assets/124381395/45d43eb0-22c0-444c-bcbb-8e6615cab6bf)

Herramienta de "Detalles de acceso", no devuelve los detalles para el alumno
![imagen](https://github.com/chamilo/chamilo-lms/assets/124381395/d4558205-a906-4552-9767-1b74b7f63308)

**Posible solución**
Agregar una nueva condición de acceso a la información en [main/inc/ajax/myspace.ajax.php](https://github.com/chamilo/chamilo-lms/blob/1.11.x/main/inc/ajax/myspace.ajax.php#L13-L14) para que los usuarios tutores generales de una sesión (is_session_general_coach) puedan acceder a la información de acceso de los alumnos, por ejemplo:

![imagen](https://github.com/chamilo/chamilo-lms/assets/124381395/a6619e31-829b-4437-8d3c-919d87022752)

Con este cambio el tutor general podría ver los detalles de la actividad de acceso:

![imagen](https://github.com/chamilo/chamilo-lms/assets/124381395/af1735a5-8fab-49c1-b84f-80dc09083fb4)

**Comment by ywarnier** [comment]
Estoy de acuerdo con tu propuesta de añadir api_is_session_general_coach() a la lista.

**Comment by juancp-contidosdixitais** [comment]
Publicado pull request
